frontend/src/metabase/lib/formatting/numbers.tsx

```diff
@@ -141,7 +142,7 @@ export function formatNumber(
       // fall back to old, less capable formatter
       // NOTE: does not handle things like currency, percent
       return FIXED_NUMBER_FORMATTER(
-        d3.round(number, options.maximumFractionDigits),
+        roundFloat(number, options.maximumFractionDigits),
```

Comment:
`d3.round` has been removed with no alternatives, so we have to write this ourselves.